main.c

```diff
@@ -138,7 +143,12 @@ static comms_layer_t * radio_setup (am_addr_t node_addr, uint8_t eui[IEEE_EUI64_
     debug1("radio rdy");
     return radio;
 }
+#ifdef INCLUDE_OTA
+static void ota_active(bool status)
+{
```

Comment:
I think it is necessary to do something here. When libbeat is included, dissable it temporarily?